source/documentation/deploying_apps/buildpacks.md

```diff
@@ -12,7 +12,7 @@ Cloud Foundry uses buildpacks to provide runtime and framework support for appli
 
 We recommend using the standard buildpacks to maximise the support you will receive from GOV.UK PaaS. Using a custom buildpack or Docker image will mean additional setup and maintenance costs.
 
-Refer to the [Guidance](https://docs.cloud.service.gov.uk/#guidance-buildpacks) section for further information on the different options available.
+Your responsibilities change depending on whether you use a standard buildpack, custom buildpack or a docker image. Refer to the [responsibility model guidance](/#responsibility-model) for further information.
```

Comment:
capitalisation: `Docker`

Reply:
This is interesting, I recall some discussions that "Docker" with upper case refers to the company and the product, then use "docker image"  "docker network", etc to refer to the tech stack.

But I am happy to change it.